Core/Core/Data/Model/Data_Dashboard.swift

```diff
@@ -112,9 +111,6 @@ public extension DataLayer {
         public let courseImage: String
         public let courseAbout: String
         public let courseSharingUtmParameters: CourseSharingUtmParameters
-        public let courseUpdates: String
-        public let courseHandouts: String
-        public let discussionURL: String
```

Comment:
Consider this idea: If we render all fields optional, it would prevent any disruptions in functionality, even if certain fields remain unused or no data is received from the backend.